bin/update-package-references

```diff
@@ -19,11 +19,16 @@ DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
 
```

Comment:
This file is run in `master.yml` right now on merges to master.  Its job is to update package.json and Gemfile references to etna to specific commits to the monorepo instead.  This removes the need to upload to npm or ruby gems to issue a deploy.

Reply:
This will also, ALL GO AWAY once deployments switch to using docker itself (since the images will have already bundled etna alongside the code).  This exists primarily as a gap to allow our current deployment process to continue to run naively while we get the docker foundations in place.